.storybook/stories/paragraphs/Gallery.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import type {Meta, StoryObj} from '@storybook/react';
+import {ComponentProps} from "react";
+import GalleryParagraph from '@components/paragraphs/stanford-gallery/gallery-paragraph';
+import { getStoryBookGalleryImage } from '../storybook-entities';
+
+type ComponentStoryProps = ComponentProps<typeof GalleryParagraph> & {
+
+}
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
+const meta: Meta<ComponentStoryProps> = {
+  title: 'Design/Paragraphs/Gallery',
+  component: GalleryParagraph,
+  tags: ['autodocs'],
+  argTypes: {
+    numberofimages: {
```

Comment:
@pookmish Is this the kind control you were talking about for the stories? I was thinking that it would be nice for the authors to see what it would look like to try different number of images.  What do you think?

Reply:
That's definitely one way to go about it. you could change this from a select to a `number` though. Then it would just create as many as the user would like to see.

See the [storybook documentation](https://storybook.js.org/docs/api/arg-types#control) for more.